From f5e19d117f2c178c79b964fd546c6c1594096b95 Mon Sep 17 00:00:00 2001
From: Chao Zeng <chao.zeng@siemens.com>
Date: Mon, 13 Sep 2021 09:21:50 +0800
Subject: [PATCH 9/9] u-boot:change the boot order

Bypass and start directly from emmc. In case the user cannot switch
the boot medium after falling into industry os

Signed-off-by: Chao Zeng <chao.zeng@siemens.com>
---
 include/configs/iot2050.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/configs/iot2050.h b/include/configs/iot2050.h
index ddb4cfcc8e..250f87ef42 100644
--- a/include/configs/iot2050.h
+++ b/include/configs/iot2050.h
@@ -44,9 +44,9 @@
  * runtime, when that board was detected.
  */
 #define BOOT_TARGET_DEVICES(func) \
-	func(MMC, mmc, 1) \
+	BOOT_TARGET_USB(func) \
 	func(MMC, mmc, 0) \
-	BOOT_TARGET_USB(func)
+	func(MMC, mmc, 1) 
 
 #include <config_distro_bootcmd.h>
 
-- 
2.32.0

